fuzz/src/chanmon_consistency.rs

```diff
@@ -532,6 +526,7 @@ pub fn do_test(data: &[u8]) {
 							events::MessageSendEvent::SendChannelReestablish { .. } => {},
 							events::MessageSendEvent::SendFundingLocked { .. } => {},
 							events::MessageSendEvent::PaymentFailureNetworkUpdate { .. } => {},
+							events::MessageSendEvent::HandleError { .. } => {},
```

Comment:
Same here - this indicates exactly the type of failure this fuzz target exists to catch.